core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -465,11 +530,13 @@ else if (cmp == 0) {
          */
         final long valueReference;
         final int entryIndex;
+        int version;
```

Comment:
Excellent comments about slice and version, please explain about version as well....